geppetto.js/geppetto-ui/src/3d-canvas/threeDEngine/ThreeDEngine.js

```diff
@@ -682,6 +682,10 @@ export default class ThreeDEngine {
   setupListeners (onSelection) {
     const that = this;
 
+    this.controls.removeEventListener('start', this.requestFrame());
```

Comment:
I don't expect this to work: a function was added and the same function needs to be removed (`this.requestFrame()` is not even a function). The function was defined inline and nothing references it, so it should be saved somewhere and retrieved back to be removed.
Also, missing some listeneners removals, like mouseOver